index.d.ts

```diff
@@ -979,15 +979,12 @@ export interface EmptyStateOwnProps {
   background?: 'light' | 'dark'
   /** the primary cta of the empty state */
   primaryCta?: React.ReactNode
-  /** the secondary cta of the empty state */
-  annchorCta?: React.ReactNode
```

Comment:
Oh my! Let's change this `anchorCta`

Reply:
oh wow I didn't notice this after reading it for the 5th time. good catch. thanks!